src/Compilers/CSharp/Portable/Symbols/Tuples/TupleFieldSymbol.cs

```diff
@@ -67,22 +61,35 @@ public override bool IsDefaultTupleElement
             }
         }
 
-        public override FieldSymbol TupleUnderlyingField
+        public sealed override FieldSymbol TupleUnderlyingField
         {
             get
             {
                 return _underlyingField;
             }
         }
 
-        public override Symbol AssociatedSymbol
+        public override Symbol? AssociatedSymbol
         {
             get
             {
                 return _containingTuple.GetTupleMemberSymbolForUnderlyingMember(_underlyingField.AssociatedSymbol);
             }
         }
 
+        public override FieldSymbol OriginalDefinition
+        {
+            get
+            {
+                NamedTypeSymbol originalContainer = ContainingType.OriginalDefinition;
+                if (!originalContainer.IsTupleType || ContainingType.IsDefinition)
```

Comment:
>originalContainer.IsTupleType [](start = 21, length = 29)

It looks like the constructor asserts this is true.

Reply:
The OriginalDefinition of a long tuple is not a tuple type. It is `VT'8`.

---
In reply to: [355692545](https://github.com/dotnet/roslyn/pull/39370#discussion_r355692545) [](ancestors = 355692545)

Reply:
Is that true only when it's a long tuple?

---
In reply to: [355781348](https://github.com/dotnet/roslyn/pull/39370#discussion_r355781348) [](ancestors = 355781348,355692545)